ols/app/endpoints/ols.py

```diff
@@ -584,6 +586,8 @@ def store_conversation_history(
                 response_message.response_metadata["provider"] = llm_request.provider
             if llm_request.model:
                 response_message.response_metadata["model"] = llm_request.model
+            if rag_chunks:
+                response_message.additional_kwargs["reference_documents"] = build_referenced_docs(rag_chunks)
```

Comment:
I think we should name this `referenced_documents` to be consistent with streaming API response.

```suggestion
                response_message.additional_kwargs["referenced_documents"] = build_referenced_docs(rag_chunks)
```

Reply:
updated